storm-core/src/jvm/org/apache/storm/trident/graph/Group.java

```diff
@@ -65,6 +68,25 @@ public Group(Group g1, Group g2) {
     }
 
     @Override
+    public Map<String, Number> getResources() {
+        Map<String, Number> ret = new HashMap<>();
+        for(Node n: nodes) {
+            if(n instanceof ITridentResource) {
+                Map<String, Number> res = ((ITridentResource)n).getResources();
```

Comment:
minor: n is always an instance of `ITridentResource`, no need of if check and type casting.